packages/mantine/src/components/table/table-actions/TableActionContext.ts

```diff
@@ -0,0 +1,9 @@
+import {createSafeContext} from '@mantine/core';
+
+export interface TableActionContextValue {
+    primary: boolean;
+}
+
+export const [TableActionProvider, useTableActionContext] = createSafeContext<TableActionContextValue>(
```

Comment:
Where is this used?

Reply:
In the action to determine if it's primary or secondary: https://github.com/coveo/plasma/pull/3725/files#diff-fc82ca754653dc5b838fec56691ed3fe2538f00d1a544cb7d0b8d5b28fc61c96R43

And in the ActionList to wrap each section with the correct provider:
- https://github.com/coveo/plasma/pull/3725/files#diff-45d043921d99311e7caff64898eade5624e8c3ee09f64e8166eb3713146993d6R109-R111
- https://github.com/coveo/plasma/pull/3725/files#diff-45d043921d99311e7caff64898eade5624e8c3ee09f64e8166eb3713146993d6R140

It really simplified the code